recipes/shapeshifter/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "shapeshifter" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dea83a20033b75e35cd4483b8703d322f81315a448133615f9254d75a03c1924
+
+build:
+  skip: True # [py27]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pandas
```

Comment:
- pip
- python >=3

and the rest can be dropped from the host requirements imho.